.travis/setup_cassandra.sh

```diff
@@ -5,5 +5,5 @@ source ./versions.sh
 CASSANDRA_BASE=apache-cassandra-$CASSANDRA_VERSION
 
 sudo rm -rf /var/lib/cassandra/*
-curl http://www.us.apache.org/dist/cassandra/$CASSANDRA_VERSION/$CASSANDRA_BASE-bin.tar.gz | tar xz
+curl http://apache.spinellicreations.com/cassandra/$CASSANDRA_VERSION/$CASSANDRA_BASE-bin.tar.gz | tar xz
```

Comment:
It is official, it is a mirror on the official download page. This change shouldn't be here anyways, oops.